samples/net/sockets/echo_server/boards/nucleo_g431rb.conf

```diff
@@ -0,0 +1,52 @@
+# Generic networking options
+CONFIG_NETWORKING=y
+CONFIG_NET_UDP=y
+CONFIG_NET_TCP=n
+CONFIG_NET_IPV6=n
+CONFIG_NET_IPV4=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+CONFIG_POSIX_MAX_FDS=4
+CONFIG_NET_CONNECTION_MANAGER=y
+
+# Kernel options
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_ENTROPY_GENERATOR=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_INIT_STACKS=y
+
+# Logging
+CONFIG_NET_LOG=n
+CONFIG_LOG=n
+CONFIG_NET_STATISTICS=n
+CONFIG_PRINTK=n
+
+# Network buffers
+CONFIG_NET_PKT_RX_COUNT=4
+CONFIG_NET_PKT_TX_COUNT=4
+CONFIG_NET_BUF_RX_COUNT=24
+CONFIG_NET_BUF_TX_COUNT=24
+CONFIG_NET_CONTEXT_NET_PKT_POOL=y
+
+# IP address options
+CONFIG_NET_MAX_CONTEXTS=2
+CONFIG_NET_MAX_CONN=2
+
+# Network shell
+CONFIG_NET_SHELL=n
+CONFIG_SHELL=n
+
+# Network application options and configuration
+CONFIG_NET_CONFIG_SETTINGS=y
+CONFIG_NET_CONFIG_NEED_IPV4=y
+CONFIG_NET_CONFIG_MY_IPV4_ADDR="192.168.3.3"
+CONFIG_NET_CONFIG_PEER_IPV4_ADDR="192.168.3.1"
+
+# How many client can connect to echo-server simultaneously
+CONFIG_NET_SAMPLE_NUM_HANDLERS=1
+
+# LAN865x T1S driver specific
+CONFIG_NET_L2_ETHERNET=y
+CONFIG_NET_PROMISCUOUS_MODE=y
```

Comment:
Why would we need to enable promiscuous mode by default?

Reply:
That was a leftover from testing - I will comment this out (to show that LAN865x supports it)